pages/common/wfuzz.md

```diff
@@ -0,0 +1,20 @@
+# wfuzz
+
+> A web application bruteforcer.
+> More information: <https://wfuzz.readthedocs.io/en/latest/user/basicusage.html>.
+
+- Directory and file bruteforce using the specified wordlist and also proxying the traffic:
+
+`wfuzz -w {{path/to/file}} -p {{127.0.0.1:8080}} {{http://example.com/FUZZ}}`
+
+- Save the results to a file:
+
+`wfuzz -w {{path/to/file}} -f {{filename}} {{http://example.com/FUZZ}}`
+
+- Show colorized output while only showing the declared response codes in the output:
+
+`wfuzz -c -w {{path/to/file}} --sc {{200,301,302}} {{http://example.com/FUZZ}}`
+
+- Use a custom header to fuzz subdomains while hiding specific response codes and word counts. Increase the threads to 100 and include the target ip/domain:
```

Comment:
When executing a `wfuzz` command you often have many undesirable pages in the output. Similar to, `--hc`, which hides response codes, `--hw` hides a specified number of words. 222 is an arbitrary number which would exclude all pages that include 222 words within the HTTP response. Since `wfuzz doesn't offer full word switches I tried to be as concise as possible in the descriptions. Hence, 'hiding specific response and word counts.' This is explaining '--hc {{301}} --hw {{222}}'.

Reply:
Thanks! 🚀